src/types.ts

```diff
@@ -15,9 +15,17 @@ export interface PlaybackData {
   timeUnits: TimeUnits;
 }
 
-export interface CheckboxDisplayOptions {
+export interface CheckboxDisplayOptionsHide {
   hideTooltipText: string;
+}
```

Comment:
does this need to be `hideTooltipText?` can it just be tooltip text and part of a more generic type?